test/integration/full-content/fixtures/core__search__custom-text.serialized.html

```diff
@@ -0,0 +1 @@
+<!-- wp:search /-->
```

Comment:
The fact that this markup does not match `core__search__custom-text.html` should be alarming; it's the entire purpose of these fixtures to guarantee. Running `npm run fixtures:regenerate` locally seems to produce some changes for me which produce a result more to what's expected.

However, it's still very concerning that:

- The build passes despite these markups being entirely different
- The build passes despite there being local changes to commit after running `npm run fixtures:regenerate`

Reply:
See #14122 for fix